pdns/dnsdistdist/docs/advanced/axfr.rst

```diff
@@ -34,3 +56,14 @@ check to :program:`dnsdist`'s side::
 
 .. versionchanged:: 1.4.0
   Before 1.4.0, the RCodes were in the ``dnsdist`` namespace. Use ``dnsdist.REFUSED`` in these versions.
+
+.. warning::
+
+  Be wary of dnsdist caching the responses to NOTIFY queries and sending these to the wrong clients.
+  This is mitigated by default when the source IP of the client is passed using EDNS Client Subnet, but
+  not when the proxy protocol is used, so disabling caching for these kinds of queries is advised:
+
+  .. code-block:: lua
+
+    -- this rule will disable caching for NOTIFY responses
```

Comment:
```suggestion
    -- this rule will disable caching for NOTIFY and SOA responses
```

(assuming that was intentional)

Reply:
Would `this rule will disable the caching of responses for NOTIFY queries with a SOA qtype` be more clear, perhaps? As far as I know there is no other supported `qtype` for `NOTIFY` queries, but I might have missed one RFC or two or twenty ;-).

Reply:
oh sorry, it's `AndRule`! I think you can just leave it. We could also play it extra safe and ignore the qtype?

Reply:
I'm less sure about hardcoding it for NOTIFY, although I can't argue why very well.

Reply:
Alright, let's not, then, it's easy to add a rule if needed!

Reply:
:+1: